src/util/arrayView.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {type PluckFn} from "./sortBy";
+import sortBy from "./sortBy";
+
+export class ArrayView<T> {
```

Comment:
Another thing: Let's have a module docstring explaining what the intention is for this module. The context that it's specifically intended for displaying data in frontends would be helpful.

Since it's intended for frontend use only, we should probably put it in `src/webutil` instead of `src/util`

Reply:
sure, will move it and add a doc comment.